website/docs/docs/use-dbt-semantic-layer/quickstart-sl.md

```diff
@@ -112,13 +116,11 @@ User data passes through the Semantic Layer on its way back from the warehouse.
 </details>
 <details>
 <summary>Is the dbt Semantic Layer open source?</summary>
-The dbt Semantic Layer is proprietary, however, some components of the dbt Semantic Layer are open source, like dbt-core and MetricFlow. <br /><br />The universal dbt Semantic Layer is available to all Team and Enterprise Plans during public beta. Users on dbt Cloud Developer plans or dbt Core users can use MetricFlow to only define and test metrics locally.</details>
+The dbt Semantic Layer is proprietary, however, some components of the dbt Semantic Layer are open source, like dbt-core and MetricFlow. <br /><br />dbt Cloud Developer or dbt Core users can define metrics in their project, including a local dbt Core project, using the dbt Cloud IDE or the MetricFlow CLI.  However, to experience the universal dbt Semantic Layer and access those metrics using the API or downstream tools, users will must be on a dbt Cloud <a href="https://www.getdbt.com/pricing/">Team or Enterprise plan.</a></details>
```

Comment:
Same feedback (as further up) regarding MetricFlow CLI availability for dbt Cloud Developer users. If you scroll up to line 38 in this same doc, it only mentions dbt Core.